### PR TITLE
Restructured Base Harvest

### PR DESCRIPTION
**Pull request by aruzicka555**
Restructured Base Harvest in such a way that a user can get all the required and most recent DLLs for developing Base-Harvest by simply building the solution, no other work required. Removed some unnecessary files and removed the need for the use of an SDK.

### CODE REVIEW

**Comment by bmarron18** [comment]
Excellent work Austen! I am merging this pull request and will subsequently test the viability of the changes from the 'scientist friendly' perspective.

File: src/base-harvest.csproj.user [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+  </PropertyGroup>
+  <PropertyGroup>
+    <ReferencePath>C:\Users\Austen\Documents\GitHub\Extension-Base-Harvest\src\libs\</ReferencePath>
```
Comment (bmarron18) [inline src/base-harvest.csproj.user]: I assume that the reference path will change per user after opening the .csproj file in VisualStudio

**Comment by aruzicka555** [comment]
Bruce,

I had hoped it would but looks like not. I can spend more time trying to
figure out relative reference paths in VS, or you can just remove that
reference since it only works on my computer and add an instruction that
the user has to add the path themselves.

-Austen

**Comment by bmarron18** [comment]
Yes, I don't see a real problem with a slight user input. Thanks Austen.

**Pull request merged**